media/css/firefox/developer-quantum.scss

```diff
@@ -0,0 +1,596 @@
+                clear: left;
+            }
+        }
+    }
+}
+
+@supports (display: grid) {
+    @media #{$mq-phone-wide} {
+        .gallery-list {
+            display: grid;
+            grid-gap: 40px 20px;
+            grid-template-columns: repeat(2, 1fr);
+
+            // undo the clearfix so it doesn't bork the grid
+            &:after {
+                content: none;
+            }
+
+            .gallery-item {
+                clear: none;
+                float: none;
+                margin: 0;
+                padding: 0;
+                width: auto;
+            }
+        }
+    }
+
+    @media #{$mq-tablet} {
+        .gallery-list {
+            grid-template-columns: repeat(3, 1fr);
+        }
+    }
+}
+
+
+//*------------------------------------------------------------------*/
+// Engagement
+.section-engage {
+    background: $color-dev-mediumblue linear-gradient(120deg, $color-dev-lightblue 0, $color-dev-mediumblue 100%) no-repeat;
+    color: #fff;
+    text-align: center;
+
+    h3 {
+        @include font-size(24px);
+        margin: 0 0 20px;
+    }
+
+    .speak-up {
+        border-bottom: 1px solid #fff;
+        margin-bottom: 40px;
+        padding-bottom: 40px;
+    }
+
+    @media #{$mq-tablet} {
+        padding: 160px 20px 40px;
+
+        h3 {
+            @include font-size-level3;
+        }
+
+        .speak-up,
+        .get-involved {
+            @include border-box;
+            @include span(6);
+            padding: 20px 40px;
+        }
+
+        .speak-up {
+            border-bottom: 0;
+            margin-bottom: 0;
+            padding-bottom: 20px;
+        }
+
+        .get-involved {
+            border-left: 1px solid #fff;
+        }
+    }
+
+    a.button.button-hollow {
+        @include font-size-level5;
+        border-color: #fff;
+        border-radius: 100px;
+        color: #fff;
+        margin-top: 20px;
+
+        &:hover,
+        &:focus,
+        &:active {
+            background-color: rgba(255, 255, 255, .1);
+            border-color: #fff;
+            color: #fff;
+        }
+    }
+}
+
+
+//*------------------------------------------------------------------*/
+// Download
+.section-download {
+    text-align: center;
+
+    .section-body {
+        @include border-box;
+        background: url('/media/img/logos/firefox/logo-developer-quantum.png') center 20px no-repeat;
+        @include background-size(165px 165px);
+        max-width: $width-desktop;
+        padding-top: 220px;
+    }
+
+    h2 {
+        @include font-size-level3;
+        font-weight: normal;
+        margin: 0 0 1.5em;
+    }
+
+    @media #{$mq-tablet} {
+        padding: 40px 0;
+        text-align: left;
+
+        &:after {
+            @include background-size(3000px 62px);
+            @include section-curve;
+            background-image: url('/media/img/firefox/products/developer/quantum/curve-download-bottom.svg');
+            height: 62px;
+            bottom: -62px;
```

Comment:
This leaves a thin black border between the nightly down and the newsletter section. Changing this to `bottom: -61px;` solves the problem.